he.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+
+"SEARCH_PLACEHOLDER" = "חיפוש";
+
+"ALERT_OK" = "אישור";
+"ALERT_ADD" = "הוספה";
+"ALERT_CANCEL" = "ביטול";
+"ALERT_DELETE" = "מחיקה";
+
+"ADD_STATION_WINDOW_TITLE" = "הוסף תחנה חדשה";
+"EDIT_STATION_WINDOW_TITLE" = "ערוך תחנה";
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "שם";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://path/to/stream";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "בחר תמונה";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "הוסף אוסף חדש";
+"EDIT_COLLECTION_WINDOW_TITLE" = "ערוך אוסף";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "האוסף שלי";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "ערוך";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "מחק אוסף";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "שתף";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "ערוך";
+"STATION_CONTEXT_DELETE_COLLECTION" = "מחק תחנה";
+"STATION_CONTEXT_MOVE" = "העבר לאוסף";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "מחק את \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "האם אתה בטוח שברצונך למחוק את האוסף הזה ואת כל התחנות שבתוכו?";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "מחק את \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "האם אתה בטוח שברצונך למחוק את התחנה הזאת?";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "אין תחנות";
+
+"WELCOME_COPY" = "ברוכים הבאים ל־Broadcasts! אם תרצו, יש לנו כמה תחנות ביתיות מוגדרות מראש שנוכל להוסיף לספרייה שלכם בתור התחלה.\n\nאו, אם תעדיפו, הוסיפו תחנות משלכם באופן ידני או בחרו מתןך אלפי תחנות במדריך.";
+
+"WELCOME_UPSELL" = "ברוכים הבאים ל־Broadcasts! בגרסה החינמית הזאת, יש לנו כמה תחנות ביתיות מוגדרות מראש שנוכל להוסיף לספרייה שלכם בתור התחלה.\n\nאו, אם תעדיפו לשדרג לגרסה בתשלום, תוכלו להוסיף תחנות משלכם באופן ידני או לבחור מתוך אלפי תחנות במדריך.";
+
+"WELCOME_ADD_PRESETS" = "הוסף תחנות מוגדרות מראש";
+"WELCOME_NO_THANKS" = "לא תודה";
+
+"GENERATED_COLLECTION_TITLE" = "האוסף שלי";
+
+"UPSELL_COPY" = "תודה על השימוש ב־Broadcasts!\n\nתוכלו לפתוח את התכונות המתקדמות של Broadcasts באמצעות רכישה בתוך האפליקציה, אם תרצו.\n\nאו, אם תעדיפו, המשיכו להשתמש ולערוך את התחנות המוגדרות מראש!";
+"UPSELL_BUY" = "רכוש את (%@)";
+"UPSELL_NOT_AVAILABLE" = "לא זמין";
+"UPSELL_RESTORE" = "שחזר רכישה";
+"UPSELL_NO_THANKS" = "לא תודה";
+
+"TOOLBAR_NEW_CATEGORY_TOOLTIP" = "אוסף חדש";
+"TOOLBAR_ADD_STATION_TOOLTIP" = "הוסף תחנה";
+"TOOLBAR_SHOW_SOURCES_TOOLTIP" = "הראה אוספים";
+"TOOLBAR_TOGGLE_SIDEBAR_TOOLTIP" = "הצג/הסתר את סרגל הצד";
+"TOOLBAR_TOGGLE_VIEW_TOOLTIP" = "שנה תצוגה";
+
+"IMAGE_WELL_REMOVE" = "הסר";
+"IMAGE_WELL_PASTE" = "הדבק";
+"IMAGE_WELL_COPY" = "העתק";
+
+"CARPLAY_EMPTY_LIBRARY" = "אין אוספים";
+"CARPLAY_ALL_STATIONS" = "כל התחנות";
+
+"MAC_ADD_STATION_BUTTON" = "הוסף תחנה";
+"MAC_EDIT_STATION_BUTTON" = "ערוך תחנה";
+
+"MAC_ADD_COLLECTION_BUTTON" = "הוסף אוסף";
+"MAC_EDIT_COLLECTION_BUTTON" = "ערוך אוסף";
+
+"STATION_DIRECTORY_WINDOW_TITLE" = "סייר התחנות";
+"STATION_DIRECTORY_COUNTRIES_TITLE" = "מדינות";
+"STATION_DIRECTORY_STATIONS" = "תחנת %@";
+"STATION_DIRECTORY_STATIONS_PLURAL" = "תחנות %@";
```

Comment:
In RTL languages, the stations count should display before the translation  ("xxx stations" instead of  "stations xxx").
![81504638-57b58700-92e2-11ea-920a-477fe5fc3878_LI](https://user-images.githubusercontent.com/21154194/81506807-ade5f280-9301-11ea-8282-e22978835d6d.jpg)